packages/api/cms-api/src/generator/generate-crud-input.ts

```diff
@@ -194,7 +194,7 @@ export async function generateCrudInput(
                     decorators.push("@IsString({ each: true })");
                 } else if (refType == "integer") {
                     type = "number[]";
-                    decorators.push("@Transform(({ value }) => value.map(id) => parseInt(id))");
```

Comment:
@nsams this looks like a syntax error to me. I tried to fix it, but please verify if my change is needed.